ethereum/referencetests/build.gradle

```diff
@@ -100,6 +133,7 @@ dependencies {
   referenceTestImplementation project(path: ':evm')
   referenceTestImplementation project(path: ':testutil')
   referenceTestImplementation project(path: ':util')
+  referenceTestImplementation 'ethereum:execution-spec-tests:0.2.3:fixtures@tar.gz'
```

Comment:
Can the version be included in the versions.gradle file?

Reply:
unfortunately not ... I did try it, but Spring's dependency management doesn't seem to work with the `revision]/[classifier].[ext]` pattern.